tests/integration-tests/configs/common/common.yaml

```diff
@@ -110,6 +110,13 @@ configure:
         - regions: ["eu-north-1"] # must be regions that do not have t2.micro
           oss: ["centos7"]
           schedulers: ["slurm"]
+  test_pcluster_configure.py::test_efa_and_placement_group:
+    dimensions:
+      - regions: ["us-east-1"]
+        instances: {{ common.INSTANCES_EFA_SUPPORT_X86 }}
```

Comment:
Shouldn't we test both the case of instances supporting EFA and instances not supporting it?

Reply:
All the other pcluster configure tests use instance types that don't support EFA so I figured we could save some time and resources by skipping this. I could add the test in order to be thorough though.

Reply:
This is not a blocking comment , but having other tests use non-EFA instances is not enough if such tests does not check for what we expect. Do they do it?